### DISCUSSION TITLE
Automatic Service Discover through docker

### DISCUSSION THREAD

**Discussion by AlexZorzi** [IceWhaleTech/CasaOS#174] [Ideas]
Automatically add the available services by checking which stacks are running, through docker directly or portainer api

**Comment by JohnGuan** [comment]
Quick Tip: 
Please revise your idea according to the **[Ideas Guidelines](https://github.com/IceWhaleTech/CasaOS/discussions/164)** so that everyone can better understand the details and participate in the discussion.
